cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -94,7 +93,7 @@ var describeClusterClusterCmd = &cobra.Command{
 
 		# Describe the cluster named test-1 disabling automatic echo suppression
         # e.g. show the infrastructure machine objects, no matter if the current state is already reported by the machine's Ready condition.
-		clusterctl describe cluster test-1 --disable-no-echo`),
+		clusterctl describe cluster test-1 --echo`),
```

Comment:
```suggestion
                # Describe the cluster named test-1 showing the MachineInfrastructure and BootstrapConfig objects 
                # also when their status is the same as the status of the corresponding machine object.
		clusterctl describe cluster test-1 --echo=true`),
```
Given that echo is a negation disable-no-echo